src/Phinx/Db/Adapter/UnsupportedColumnTypeException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Phinx\Db\Adapter;
+
+/**
+ * Exception thrown when a column type doesn't match a Phinx type.
+ *
+ * @author Martijn Gastkemper <martijn@grrr.nl>
+ */
+class UnsupportedColumnTypeException extends \Exception {
```

Comment:
Probably extending `RuntimeException` is a good idea to keep other things working as they used to